hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/DataSerializableSerializer.java

```diff
@@ -46,8 +46,13 @@
  * <li>{@link IdentifiedDataSerializable}</li>
  * </ol>
  */
+@SuppressWarnings("checkstyle:npathcomplexity")
 final class DataSerializableSerializer implements StreamSerializer<DataSerializable>, TypedStreamDeserializer<DataSerializable> {
 
+    public static final byte IDS_FLAG = 1 << 0;
+    public static final byte EE_FLAG = 1 << 1;
+    public static final byte COMP_FLAG = 1 << 2;
```

Comment:
Maybe add short comment on what are these flags. I guess IDS is IdentifiedDataSerializable, EE is Enterprise but I couldn't figure out what COMP was.